gradle-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -5,4 +5,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.2-bin.zip
```

Comment:
Could we extract the version to a variable? of the URL is stable.

Reply:
there are millions of these files in github - Gradle have not provided a tidier mechanism - I don't want to reinvent Gradle - can we wait for Gradle to make this easier instead of me spending time on it?

Reply:
is sbt any better, we have dozens of build.properties files checked in that all say `sbt.version=1.9.4`